CHARTER.md

```diff
@@ -0,0 +1,120 @@
+# Project Charter (the “Charter”) for OpenSSF Malicious Packages 
+
+***Created June 11, 2025***
+
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the OpenSSF Malicious Packages open source project, which has been established as OpenSSF Malicious Packages a Series of LF Projects, LLC (the “Project”). LF Projects, LLC (“LF Projects”) is a Delaware series limited liability company. All contributors (including committers, maintainers, and other technical positions) and other participants in the Project (collectively, “Collaborators”) must comply with the terms of this Charter.
+
+## 1. Mission and Scope of the Project
+
+- a. The mission of the Project is to collect into one centralized repository malicious packages published to the NPM, PyPI, RubyGems, Nuget, Maven, Go and Crates.io software registries. This project will act as a singular place for security researchers and vendors to publish malicious packages they find.
```

Comment:
Can you make the language less specific to a set of ecosystems.

For example "... published to open source package registries, like NPM, PyPI, RubyGems, etc."